transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/WsnConnectorTest.java

```diff
@@ -335,12 +335,8 @@ public void doMessageReceived(final IoSessionEx session, Object message)
                 IoSessionEx sessionEx = session;
                 System.out.println("Acceptor: received message: " + Utils.asString(buf.buf()));
                 IoBufferAllocatorEx<?> allocator = sessionEx.getBufferAllocator();
-                session.write(allocator.wrap(asByteBuffer("Reply from acceptor"))).addListener(new IoFutureListener<IoFuture>() {
-                    @Override
-                    public void operationComplete(IoFuture future) {
-                        session.close(true);
-                    }
-                });
+                WriteFuture writeFuture = session.write(allocator.wrap(asByteBuffer("Reply from acceptor")));
+                writeFuture.addListener(future -> session.close(false));
```

Comment:
Note session.close(true) before and session.close(false) after the change.  Otherwise the semantics of the code before and after the change are the same.